parsers/jscript.c

```diff
@@ -2005,6 +2007,9 @@ static bool parseES6Class(tokenInfo *const token,tokenInfo * const parent)
 		if(isType(token,TOKEN_CLOSE_CURLY))
 			break;
 		
+		if(isKeyword(token,KEYWORD_static))
+			readToken(token);
+		
```

Comment:
I don't know javascript, but I guess. like C language, `static` is important information.
So I wonder we can store it to a field of tag as you did in `property:` in Cxx.
This is just an optional idea.